### PR TITLE
Video: update download link for NanoTrack

### PR DESCRIPTION
**Pull request by zihaomu**
Merge this PR with: https://github.com/opencv/opencv_extra/pull/1025.

Update model download link without code change.
NanotrackV2 is faster than NanoTrackV1. At the same time, the accuracy rate is improved.

### Test at Apple M1:
| Model Version | Speed |  VOT2018 EAO (higher is better) |
|:-------:|:--------:|:------------:|
| NanoTrackV1 | 150 FPS | 0.311 |
| NanoTrackV2 | 200 FPS | 0.352 |

### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [ ] There is a reference to the original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

**Comment by asmorkalov** [comment]
It makes sense to update the model in tests too.

**Comment by zihaomu** [comment]
Hi @asmorkalov, V1 and V2 have similar structures. And totally the same pre-process and post-process.

**Comment by asmorkalov** [comment]
Performance improvement says opposite. It's not just a retrained model. There are some changes in architecture or at least channels.

**Comment by zihaomu** [comment]
The final decision, we keep both `Nanotrack` v1 and v2 models at regression test. Since the model is quite lightweight, it will not bring too much burden to CI.

**Review by asmorkalov — APPROVED** [review]
:+1:

**Comment by asmorkalov** [comment]
@zihaomu please squash the commits. Waiting for OpenCV Extra PR approve.

**Comment by alalek** [comment]
BTW, it makes sense to add performance tests of dnn for these (and other) models: https://github.com/opencv/opencv/blob/4.x/modules/dnn/perf/perf_net.cpp

(the last model was added there in 2020)

**Comment by zihaomu** [comment]
Hi @alalek, the tracking model is different. It consists of two models, the one for extracting first frame features, and the other one for extracting and locating the object.

Which model should be put in the performance test? 
The second model is the key that affects speed.

**Comment by ycxia** [comment]
@zihaomu, support nanotrackv3?
Somthing wrong when i change model to nanotrackv3.
image

How to deal? thank you

**Comment by zihaomu** [comment]
Hi @ycxia, v3 has some changes, but not much. Will try to add nanotrack v3 next week.

**Pull request merged**
